stencil-library/src/api/SubAccount.ts

```diff
@@ -0,0 +1,146 @@
+import { formatCurrency, formatDate, formatPercentage, formatTime } from '../utils/utils';
+
+export enum TagTypes {
+  neutral = 'neutral',
+  info = 'info',
+  success = 'success',
+  warning = 'warning',
+  error = 'error'
+}
+
+export enum SubAccountStatusType {
+  created = 'created',
+  submitted = 'submitted',
+  information_needed = 'information_needed',
+  enabled = 'enabled',
+  disabled = 'disabled',
+  rejected = 'rejected',
+  archived = 'archived'
+}
+
+export enum BusinessType {
+  forProfit = 'for_profit',
+  nonProfit = 'non_profit',
+  government = 'government_entity',
+  individual = 'individual'
+}
+
+export enum ForProfitStructure {
+  none = '',
+  soleProprietorship = 'sole_proprietorship',
+  singleMember = 'single_llc',
+  multiMember = 'multi_llc',
+  privatePartnership = 'private_partnership',
+  privateCorp = 'private_corporation',
+  unincorporatedAssociation = 'unincorporated_association',
+  publicPartnership = 'public_partnership',
+  publicCorp = 'public_corporation'
+}
+
+export enum NonProfitStructure {
+  incorporated = 'incorporated',
+  unincorporated = 'unincorporated'
+}
+
+export enum GovernmentEntityStructure {
+  unit = 'government_unit',
+  instrumentality = 'government_instrumentality',
+  taxExempt = 'tax_exempt_government_instrumentality'
+}
+
+export interface ApplicationFeeRate {
+  basis_point_rate: number,
+  created_at: string,
+  currency: string,
+  id: string,
+  rate_type: string,
+  transaction_fee: number,
+  updated_at: string
+}
+
+export interface ISubAccount {
+  account_type: string,
+  application_fee_rates: ApplicationFeeRate[] | [],
+  created_at: string,
+  currency: string,
+  id: string,
+  name: string,
+  payout_ready: boolean,
+  platform_account_id: string,
+  processing_ready: boolean,
+  status: SubAccountStatusType,
+  updated_at: string
+  related_accounts?: object,
+  readonly statusTagType?: TagTypes
+};
+
+export class SubAccount implements ISubAccount {
+  public account_type: string;
+  public application_fee_rates: ApplicationFeeRate[] | [];
+  public created_at: string;
+  public dateString: string;
+  public timeString: string;
+  public currency: string;
+  public id: string;
+  public name: string;
+  public payout_ready: boolean;
+  public platform_account_id: string;
+  public processing_ready: boolean;
+  public status: SubAccountStatusType;
+  public updated_at: string;
+  public related_accounts: object;
```

Comment:
@colganmeanor this one sounds like it was going to be an array of accounts

Reply:
So this was taken directly from the type declaration in customer-admin-fe - I agree that the variable name sounds that way but it appears to be typed as an `object`  in our API Docs and thats the way it is in the customer admin as well. 🤷🏻